src/AppBundle/Entity/Notice.php

```diff
@@ -359,22 +359,26 @@ public function getScreenshot(): ?string
         return $this->screenshot;
     }
 
-    public function setScreenshot(string $screenshot): void
+    public function setScreenshot(?string $screenshot): Notice
     {
         $this->screenshot = $screenshot;
+
+        return $this;
```

Comment:
Others methods aren't making a use of the fluent interface, what's you convention on this ?

Reply:
I noticed that sometimes we use it, sometimes we don’t, so I’ve been adding it to every part I work on to tend to having it everywhere …